.flutter-plugins

```diff
@@ -1,4 +1,5 @@
 # This is a generated file; do not edit or check into version control.
+integration_test=C:\\Users\\simon\\flutter\\flutter\\.pub-cache\\hosted\\pub.dartlang.org\\integration_test-1.0.1\\
```

Comment:
windows? 😮 😉

Reply:
😄 yes I know!! my laptop has some issues so I'm using the desktop at the moment (I tried to create a dual boot with ubuntu but some problems occurs, using wsl if necessary necessary)